src/commands/info/releasenotes/display.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Needed this to ensure the "helpers" were decalred before read in examples
+/* eslint-disable @typescript-eslint/member-ordering */
+
+import { Env } from '@salesforce/kit';
+import { flags, SfdxCommand } from '@salesforce/command';
+import { getString } from '@salesforce/ts-types';
+import { Messages } from '@salesforce/core';
+
+import { getInfoConfig, InfoConfig } from '../../../shared/get-info-config';
+import { getReleaseNotes } from '../../../shared/get-release-notes';
+import { getDistTagVersion } from '../../../shared/get-dist-tag-version';
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is sfdxusing the messages framework can also be loaded this way.
```

Comment:
`sfdx using`

Reply:
side comment: I've found this pretty handy https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker